.github/workflows/maestro.yml

```
@@ -0,0 +1,143 @@
+  workflow_dispatch:
+jobs:
+  maestro-tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    env:
+      APP_ID: in.koreatech.koin
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: ./.github/actions/gradle-setup
```

actionlint (1.7.11)

[error] description is required in metadata of "Gradle Setup" action at "/home/jailuser/git/.github/actions/gradle-setup/action.yml"

(action)